spaceship/lib/spaceship/client.rb

```diff
@@ -172,7 +172,7 @@ def persistent_cookie_path
       if ENV["SPACESHIP_COOKIE_PATH"]
         path = File.expand_path(File.join(ENV["SPACESHIP_COOKIE_PATH"], "spaceship", self.user, "cookie"))
       else
-        ["~/.spaceship", "/var/tmp/spaceship", "#{Dir.tmpdir}/spaceship"].each do |dir|
+        ["~/.fastlane/spaceship", "/var/tmp/spaceship", "#{Dir.tmpdir}/spaceship"].each do |dir|
```

Comment:
Do we need to continue to check in `"~/.spaceship"` so that people don't get logged out as a result of this change?